src/plots/gl2d/scene2d.js

```diff
@@ -313,27 +313,21 @@ proto.updateRefs = function(newFullLayout) {
 };
 
 proto.relayoutCallback = function() {
-    var graphDiv = this.graphDiv,
-        xaxis = this.xaxis,
-        yaxis = this.yaxis,
-        layout = graphDiv.layout;
-
-    // update user layout
-    layout.xaxis.autorange = xaxis.autorange;
-    layout.xaxis.range = xaxis.range.slice(0);
-    layout.yaxis.autorange = yaxis.autorange;
-    layout.yaxis.range = yaxis.range.slice(0);
-
-    // make a meaningful value to be passed on to the possible 'plotly_relayout' subscriber(s)
-    // scene.camera has no many useful projection or scale information
-    // helps determine which one is the latest input (if async)
-    var update = {
-        lastInputTime: this.camera.lastInputTime
-    };
+    var graphDiv = this.graphDiv;
+    var xaxis = this.xaxis;
+    var yaxis = this.yaxis;
+
+    // make a meaningful value to be passed on to possible 'plotly_relayout' subscriber(s)
+    var update = {};
+    update[xaxis._name + '.range'] = xaxis.range.slice();
+    update[yaxis._name + '.range'] = yaxis.range.slice();
+    update[xaxis._name + '.autorange'] = xaxis.autorange;
+    update[yaxis._name + '.autorange'] = yaxis.autorange;
 
-    update[xaxis._name] = xaxis.range.slice(0);
-    update[yaxis._name] = yaxis.range.slice(0);
+    Registry.call('_storeDirectGUIEdit', graphDiv.layout, graphDiv._fullLayout._preGUI, update, true);
```

Comment:
This is a bit confusing... the `true` last arg means "apply this change to the input object" (ie `gd.layout`). But in the end this is the only caller that used that argument, everyone else (among the callers that short-circuit `restyle/relayout` so need to use `_storeDirectGUIEdit`) had some other way of applying the change... so rather than making it more explicit I think I'll just :hocho: that arg and apply the change here.

Reply:
> I think I'll just 🔪 that arg and apply the change here.

done in https://github.com/plotly/plotly.js/pull/3236/commits/86b90d4e00ca98d2e0aa7d689d561548c258c7ff
Anyway only used for `pointcloud` at this point, which is mostly obsolete and partly broken but at least it's not *more* broken now...